snap/hooks/install

```diff
@@ -21,6 +22,26 @@ for service in edgex-mongo security-proxy-setup security-secrets-setup security-
     fi
 done
 
+# handle app-service-configurable's various profiles:
+# 1. ensure all the directories from app-service-configurable exist
+# 2. copy the config files from $SNAP into $SNAP_DATA
+# 3. replace the various env vars that might be in that config file with their
+#    "current" symlink equivalent
+mkdir -p "$SNAP_DATA/config/app-service-configurable"
+cd "$SNAP/config/app-service-configurable"
+find . -maxdepth 2 -type d -exec mkdir -p "$SNAP_DATA/config/app-service-configurable/"{} \;
+find ./res -name "configuration.toml" | \
```

Comment:
I let this slide in the first iteration due to pressure schedule, however I think for this PR, we really only should be copying the rules-engine profile.

Reply:
ok

Reply:
Done